%  LaTeX support: latex@mdpi.com 
%  For support, please attach all files needed for compiling as well as the log file, and specify your operating system, LaTeX version, and LaTeX editor.




%=================================================================
%% required packages
\RequirePackage{xcolor, colortbl, tikz, graphicx, nameref}



%=================================================================
\documentclass[remotesensing, aricle,submit,moreauthors]{Definitions/mdpi} 
% For posting an early version of this manuscript as a preprint, you may use "preprints" as the journal and change "submit" to "accept". The document class line would be, e.g., \documentclass[preprints,article,accept,moreauthors,pdftex]{mdpi}. This is especially recommended for submission to arXiv, where line numbers should be removed before posting. For preprints.org, the editorial staff will make this change immediately prior to posting.

%--------------------
% Class Options:
%--------------------
%----------
% journal
%----------
% Choose between the following MDPI journals:
% acoustics, actuators, addictions, admsci, adolescents, aerospace, agriculture, agriengineering, agronomy, ai, algorithms, allergies, alloys, analytica, animals, antibiotics, antibodies, antioxidants, applbiosci, appliedchem, appliedmath, applmech, applmicrobiol, applnano, applsci, aquacj, architecture, arts, asc, asi, astronomy, atmosphere, atoms, audiolres, automation, axioms, bacteria, batteries, bdcc, behavsci, beverages, biochem, bioengineering, biologics, biology, biomass, biomechanics, biomed, biomedicines, biomedinformatics, biomimetics, biomolecules, biophysica, biosensors, biotech, birds, bloods, blsf, brainsci, breath, buildings, businesses, cancers, carbon, cardiogenetics, catalysts, cells, ceramics, challenges, chemengineering, chemistry, chemosensors, chemproc, children, chips, cimb, civileng, cleantechnol, climate, clinpract, clockssleep, cmd, coasts, coatings, colloids, colorants, commodities, compounds, computation, computers, condensedmatter, conservation, constrmater, cosmetics, covid, crops, cryptography, crystals, csmf, ctn, curroncol, currophthalmol, cyber, dairy, data, dentistry, dermato, dermatopathology, designs, diabetology, diagnostics, dietetics, digital, disabilities, diseases, diversity, dna, drones, dynamics, earth, ebj, ecologies, econometrics, economies, education, ejihpe, electricity, electrochem, electronicmat, electronics, encyclopedia, endocrines, energies, eng, engproc, ent, entomology, entropy, environments, environsciproc, epidemiologia, epigenomes, est, fermentation, fibers, fintech, fire, fishes, fluids, foods, forecasting, forensicsci, forests, foundations, fractalfract, fuels, futureinternet, futureparasites, futurepharmacol, futurephys, futuretransp, galaxies, games, gases, gastroent, gastrointestdisord, gels, genealogy, genes, geographies, geohazards, geomatics, geosciences, geotechnics, geriatrics, hazardousmatters, healthcare, hearts, hemato, heritage, highthroughput, histories, horticulturae, humanities, humans, hydrobiology, hydrogen, hydrology, hygiene, idr, ijerph, ijfs, ijgi, ijms, ijns, ijtm, ijtpp, immuno, informatics, information, infrastructures, inorganics, insects, instruments, inventions, iot, j, jal, jcdd, jcm, jcp, jcs, jdb, jeta, jfb, jfmk, jimaging, jintelligence, jlpea, jmmp, jmp, jmse, jne, jnt, jof, joitmc, jor, journalmedia, jox, jpm, jrfm, jsan, jtaer, jzbg, kidney, kidneydial, knowledge, land, languages, laws, life, liquids, literature, livers, logics, logistics, lubricants, lymphatics, machines, macromol, magnetism, magnetochemistry, make, marinedrugs, materials, materproc, mathematics, mca, measurements, medicina, medicines, medsci, membranes, merits, metabolites, metals, meteorology, methane, metrology, micro, microarrays, microbiolres, micromachines, microorganisms, microplastics, minerals, mining, modelling, molbank, molecules, mps, msf, mti, muscles, nanoenergyadv, nanomanufacturing, nanomaterials, ncrna, network, neuroglia, neurolint, neurosci, nitrogen, notspecified, nri, nursrep, nutraceuticals, nutrients, obesities, oceans, ohbm, onco, oncopathology, optics, oral, organics, organoids, osteology, oxygen, parasites, parasitologia, particles, pathogens, pathophysiology, pediatrrep, pharmaceuticals, pharmaceutics, pharmacoepidemiology, pharmacy, philosophies, photochem, photonics, phycology, physchem, physics, physiologia, plants, plasma, pollutants, polymers, polysaccharides, poultry, powders, preprints, proceedings, processes, prosthesis, proteomes, psf, psych, psychiatryint, psychoactives, publications, quantumrep, quaternary, qubs, radiation, reactions, recycling, regeneration, religions, remotesensing, reports, reprodmed, resources, rheumato, risks, robotics, ruminants, safety, sci, scipharm, seeds, sensors, separations, sexes, signals, sinusitis, skins, smartcities, sna, societies, socsci, software, soilsystems, solar, solids, sports, standards, stats, stresses, surfaces, surgeries, suschem, sustainability, symmetry, synbio, systems, taxonomy, technologies, telecom, test, textiles, thalassrep, thermo, tomography, tourismhosp, toxics, toxins, transplantology, transportation, traumacare, traumas, tropicalmed, universe, urbansci, uro, vaccines, vehicles, venereology, vetsci, vibration, viruses, vision, waste, water, wem, wevj, wind, women, world, youth, zoonoticdis 

%---------
% article
%---------
% The default type of manuscript is "article", but can be replaced by: 
% abstract, addendum, article, book, bookreview, briefreport, casereport, comment, commentary, communication, conferenceproceedings, correction, conferencereport, entry, expressionofconcern, extendedabstract, datadescriptor, editorial, essay, erratum, hypothesis, interestingimage, obituary, opinion, projectreport, reply, retraction, review, perspective, protocol, shortnote, studyprotocol, systematicreview, supfile, technicalnote, viewpoint, guidelines, registeredreport, tutorial
% supfile = supplementary materials

%----------
% submit
%----------
% The class option "submit" will be changed to "accept" by the Editorial Office when the paper is accepted. This will only make changes to the frontpage (e.g., the logo of the journal will get visible), the headings, and the copyright information. Also, line numbering will be removed. Journal info and pagination for accepted papers will also be assigned by the Editorial Office.

%------------------
% moreauthors
%------------------
% If there is only one author the class option oneauthor should be used. Otherwise use the class option moreauthors.

%---------
% pdftex
%---------
% The option pdftex is for use with pdfLaTeX. If eps figures are used, remove the option pdftex and use LaTeX and dvi2pdf.

%=================================================================
% MDPI internal commands
\firstpage{1} 
\makeatletter 
\setcounter{page}{\@firstpage} 
\makeatother
\pubvolume{1}
\issuenum{1}
\articlenumber{0}
\pubyear{2022}
\copyrightyear{2022}
%\externaleditor{Academic Editor: Firstname Lastname}
\datereceived{} 
\dateaccepted{} 
\datepublished{} 
%\datecorrected{} % Corrected papers include a "Corrected: XXX" date in the original paper.
%\dateretracted{} % Corrected papers include a "Retracted: XXX" date in the original paper.
\hreflink{https://doi.org/} % If needed use \linebreak
%\doinum{}
%------------------------------------------------------------------
% The following line should be uncommented if the LaTeX file is uploaded to arXiv.org
%\pdfoutput=1

%=================================================================
% Add packages and commands here. The following packages are loaded in our class file: fontenc, inputenc, calc, indentfirst, fancyhdr, graphicx, epstopdf, lastpage, ifthen, lineno, float, amsmath, setspace, enumitem, mathpazo, booktabs, titlesec, etoolbox, tabto, xcolor, soul, multirow, microtype, tikz, totcount, changepage, attrib, upgreek, cleveref, amsthm, hyphenat, natbib, hyperref, footmisc, url, geometry, newfloat, caption


\usepackage{lineno}
\usepackage{longtable,booktabs, amsmath, textcomp, rotating, pdflscape, array, multirow, adjustbox, threeparttable, caption, hhline, xcolor, colortbl, tabularx, tikz, float}
\usepackage{fixltx2e}
\usepackage{fontspec} % for \textsubscript
% \usepackage[options]{natbib}
\setcitestyle{square, comma, numbers,sort&compress, super}
\usepackage{graphicx}
\newcommand{\plus}{\raisebox{.4\height}{\scalebox{.6}{+ }}}
\newcommand{\minus}{\raisebox{.4\height}{\scalebox{.8}{- }}}

\newcommand\textstyleStrongEmphasis[1]{\textbf{#1}}
\makeatletter
\newcommand\arraybslash{\let\\\@arraycr}


\usepackage{listings}
\lstset{
basicstyle=\small\ttfamily,
columns=flexible,
breaklines=true
}


% % %%%%% Hyperlinks
% %% Define color for citations
% \definecolor{bluecite}{HTML}{0875b7}
% % 
% \ifthenelse{\equal{\@arttype}{article}}{
% 	% \RequirePackage[unicode=true,
% 	% bookmarksopen={true},
% 	% pdffitwindow=true,
% 	% colorlinks=true,
% 	% linkcolor=black,
% 	% citecolor=black,
% 	% urlcolor=black,
% 	% hyperfootnotes=false,
% 	% pdfstartview={FitH},
% 	% pdfpagemode=UseNone]{hyperref}
% 	% }{
% \RequirePackage[unicode=true,
% 	bookmarksopen={true},
% 	pdffitwindow=true,
% 	colorlinks=true,
% 	linkcolor=bluecite,
% 	citecolor=bluecite,
% 	urlcolor=bluecite,
% 	hyperfootnotes=false,
% 	pdfstartview={FitH},
% 	pdfpagemode= UseNone]{hyperref}
% }



%=================================================================
%% Please use the following mathematics environments: Theorem, Lemma, Corollary, Proposition, Characterization, Property, Problem, Example, Examplesandtex/MDPI_template/Definitions, Hypothesis, Remark, Definition, Notation, Assumption
%% For proofs, please use the proof environment (the amsthm package is loaded by the MDPI class).

%=================================================================
% Full title of the paper (Capitalized)
\Title{}

% MDPI internal command: Title for citation in the left column
\TitleCitation{}

% Author Orchid ID: enter ID or remove command
\newcommand{\orcidauthorA}{0000-0003-4524-2364} % Add \orcidA{} behind the author's name
\newcommand{\orcidauthorB}{0000-0002-0679-4521} % Add \orcidB{} behind the author's name

% Authors, for the paper (add full first names)
 \Author{Brendan Casey $^{1,*}$\orcidA{} and Erin Bayne $^{2}$\orcidB{}}

% and Firstname Lastname $^{2,}$*


%\longauthorlist{yes}

% MDPI internal command: Authors, for metadata in PDF
\AuthorNames{Brendan Casey and Erin Bayne}

% MDPI internal command: Authors, for citation in the left column
\AuthorCitation{Casey, B.; Bayne, E.}
% If this is a Chicago style journal: Lastname, Firstname, Firstname Lastname, and Firstname Lastname.

% Affiliations / Addresses (Add [1] after \address if there is only one affiliation.)
\address{%
$^{1}$ \quad Department of Biological Sciences, University of Alberta, Edmonton, AB T6G 2R3, Canada; bgcasey@ualberta.ca\\
$^{2}$ \quad Department of Biological Sciences, University of Alberta, Edmonton, AB T6G 2R3, Canada; bayne@ualberta.ca}

% Contact information of the corresponding author
\corres{Correspondence: bgcasey@ualberta.ca Tel.: +01-780-920-1787}


% Current address and/or shared authorship
% \firstnote{Current address: Affiliation 3.} 
% \secondnote{These authors contributed equally to this work.}
% The commands \thirdnote{} till \eighthnote{} are available for further notes

%\simplesumm{} % Simple summary

%\conference{} % An extended version of a conference paper

% Abstract (Do not insert blank lines, i.e. \\) 
\abstract{}

% Keywords
\keyword{avian; forestry; LiDAR; boreal forest} 

% The fields PACS, MSC, and JEL may be left empty or commented out if not applicable
%\PACS{J0101}
%\MSC{}
%\JEL{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Diversity
%\LSID{\url{http://}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Applied Sciences:
%\featuredapplication{Authors are encouraged to provide a concise description of the specific application or a potential application of the work. This section is not mandatory.}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Data:
%\dataset{DOI number or link to the deposited data set in cases where the data set is published or set to be published separately. If the data set is submitted and will be published as a supplement to this paper in the journal Data, this field will be filled by the editors of the journal. In this case, please make sure to submit the data set as a supplement when entering your manuscript into our manuscript editorial system.}

%\datasetlicense{license under which the data set is made available (CC0, CC-BY, CC-BY-SA, CC-BY-NC, etc.)}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Toxins
%\keycontribution{The breakthroughs or highlights of the manuscript. Authors can write one or two sentences to describe the most important part of the paper.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Only for the journal Encyclopedia
%\encyclopediadef{Instead of the abstract}
%\entrylink{The Link to this entry published on the encyclopedia platform.}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
LiDAR has the potential to improve bird models by providing high resolution structural covariates which, when paired with bird monitoring data, can provide insight into bird-habitat relationships {[}REF{]}. However, LiDAR acquisitions do not always coincide temporally with bird surveys, and it is unclear how much these time lags influence models. Disturbance-succession cycles change vegetation structure. Eventually, LiDAR metrics will no longer reflect ground conditions, and their usefulness as explanatory variables will degrade. Here, we evaluated how the time lag between LiDAR acquisitions and bird surveys influenced model robustness for early successional, mature forest, and generalist birds.

The composition and structure of forests are changing in response to climate change, shifts to natural disturbance regimes, and increasing industrial development \citep{Brandt2013}. Predictive models linking distribution, abundance, and community structure to select environmental variables have been used to understand how forest birds respond to these changes \citep{Carrillo-Rubio2014, Engler2017a, guisanPredictiveHabitatDistribution2000, He2015}. Broadly known as species distribution models(SDMs), this family of statistical methods relate field observations (e.g.~occupancy or abundance) with spatial covariates \citep{Guisan2005}. Find correlations between detection data and eviromnetal covariates to predict\ldots{} While methods vary, SDMs predict species occurrence by comparing the habitat where individuals have been observed (via traditional human point counts or autonomous bioacoustic monitoring) against habitat where species are absent {[}Guisan2005{]}. With innovations in modelling methods, computational power, and environmental monitoring, SDMs are an important tool in ecology, conservation biology, and wildlife management \citep{Elith2009}. SDMs and resulting predictive distribution maps are often applied to bird research \citet{englerAvianSDMsCurrent2017}, and have informed conservation management planning, environmental impact assessments{[}REF{]}, and bird diversity modelling {[}REF{]}\citep{englerAvianSDMsCurrent2017, franklinMappingSpeciesDistributions2010}.

\begin{center}\rule{0.5\linewidth}{0.5pt}\end{center}

See reviews by Engler et al. -\citet{englerAvianSDMsCurrent2017} and Elith et al. -\citet{Elith2009}. Engler et al. -\citet{englerAvianSDMsCurrent2017} presented a detailed overview of the many applications of SDMs in the study of birds.

They provide insite into:
1. habitat characteristics necessary for birds,
2. Predictions of distributions and forecast distributions under changing scenarios.
3. Used to estimate species abundance and density.
Explore the patterns and processes driving distributions of species in space.

Species distribution models can be used to link occurrence data with environmental covariates to predict species distributions.

While SDMs encompass a variety of statistical approaches including Generalized linear models, generalized additive models, and Bayesian approaches. MOST often encorpprate presence-absence data from species detection data from traditional human point counts or autonomous bioacoustic monitoring. Most follow the same

Many factors influence the predictive capacity of SDMs, but the inclusion of ecologically relevant spatial covariates are key drivers of model accuracy \citep{Franklin1995, Vaughn2003, fourcadePaintingsPredictDistribution2018}. Bird SDMs often rely on categorical predictors derived from digital maps delineating land cover, vegetation composition, and human footprint. While these models are often supplemented with continuous bioclimatic variables, they often miss important features driving bird response {[}REF{]}.
Continuous predictors from remote sensing can better describe the mechanisms driving habitat selection \citep{heWillRemoteSensing2015}. For example, spectral metrics linked to vegetation are often representative of the shelter and food resources used by birds. The normalized vegetation index (NDVI) derived from Sentinel and Landsat satellites has been used to measure vegetation productivity, habitat variability, and plant phenology \citep{pettorelliNormalizedDifferenceVegetation2011}. And the normalized burn ratio (NBR) can quantify disturbance severity and successional recovery rates \citep{hislopUsingLandsatSpectral2018}. MODIS land surface temperature (LST) has been used to predict bird response to heatwaves \citep{albrightHeatWavesMeasured2011}.

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 0\tabcolsep) * \real{0.9306}}@{}}
\toprule()
\begin{minipage}[b]{\linewidth}\raggedright
maps and digital forest resource inventories (FRIs) supplement bioclimatic variables; les. While these products may contain useful metrics related to plant species composition and human footprint, and canopy height, they often lack data on three-dimensional often lack more detailed measures of important habitat features driving bird habitat selection. They don't fully describe the mechanisms driving habitat selection. Lack data
Predictors from space-based optical remote sensors are used most often and can provide more detailed levels of eco improve on the class based habitat predictors of FRI \citep{heWillRemoteSensing2015}.
and extreme weather events {[}Space-based land surface temperature and precipitation measurements from MODIS have been used over terrestrial weather station data for \_models {[}REF{]}.
has been used to extract predictors related to harvest and burn severity.while spectral indices derived from satellite and aerial imagery can map land cover change and measure rates of post-disturbance habitat recovery \citep{Northrup2019, Rittenhouse2010}
\end{minipage} \\
\midrule()
\endhead
In the boreal, fire, forestry harvests, and linear features like roads and seismic lines effect forest composition, structure, and age.
froim one driven by fire to one dominated by forestry and energy exploration. composition and structure of boreal forests are changing in response to climate change, shifts to Natural disturbance regimes are being replaced by forestry and industrial development \\
Timelag between the Uderstanding the temporal limitations of LiDAR is important when selecting sources of environmental covariates. \\
Here, we evaluated how the time lag between LiDAR acquisitions and bird surveys influence the performance of SDMs across a gradient of 0 to 15 years. We evaluated if the influence of time-lag varies between early successional, mature forest, and forest generalist birds. Finally, we assessed how differences in the resultant predictive distribution maps correspond to forest age. \\
We hypothesized that the performance of models will decrease with increased LiDAR acquisition time-lag and that the magnitude of change will vary according the habitat preferences of the study species {[}figure{]}. \\
\bottomrule()
\end{longtable}

We predicted that (1) SDMs for early successional species will see the greatest declines in performance due to more rapid amounts of change in their preferred habitat during the time lag period and models will no longer be reliable after 5 years lag. (2) Forest generalist species will see moderate declines in performance as time lag increases (3) The effect of time lag on SDMs for mature forest species will be smaller due the the relative stability of their forest habitats, but will still observe mild decreases in performance as windfall and disease opens new gaps in the mature forest canopy. For all species, differences in predictive maps will be negatively correlated with the age of the forest when LiDAR was acquired.

Our methodological workflow is illustrated in Figure \ref{fig:workflow}. Analyses were done using R statistical software \citep{R-base}. We developed SDMs following the methodology outlined by Guisan and Thuiller -\citet{Guisan2005} with bird data collected by the Calling Lake Fragmentation project.

The study area is \(\approx\) 11,327 ha of boreal mixedwood forests near Calling Lake, in northern Alberta, Canada (55º14'51'\,' N, 113º28'59'\,' W) Figure \ref{fig:studyArea}). Calling Lake is located in the Boreal Central Mixedwood Natural Subregion of Northern Alberta \citep{Downing2006}. When the point counts were conducted the area contained black spruce (\emph{Picea mariana}) fens and mixedwood forests composed of trembling aspen (\emph{Populus tremuloides}, balsam poplar (\emph{Populus balsamifera}), and white spruce (\emph{Picea glauca}), with alder (\emph{Alnus spp.}) and willow (\emph{Salix spp.}) dominated understories \citep{Schmiegelow1997}. Experimental forest harvesting done through the Calling Lake Fragmentation Project has left a patchwork of successional stages amidst tracts of unharvested forest (\citet{Schmiegelow1997}). left a patchwork of forests of different stages of recovery alongside unharvested tracks (\citet{Schmiegelow1997}).

Data Sources
Point count data from the Calling Lake Fragmentation Study was accessed through the Boreal Avian Modelling Project's (BAM) avian database {[}REF{]}. Point count locations have had sixteen years of consecutive years of point count surveys, providing an opportunity to test the effects of LiDAR acquisition time lag on SDMs. See Schmiegelow et al. -\citet{Schmiegelow1997} for details on the Calling Lake Fragmentation Project's study design. Summer bird surveys were conducted annually between 1995 and 2014. Point count durations were five minutes. Within a radius of one hundred meters. Morning point counts (sunrise-10:00) 3-5 between 16 May and 7 July. We used detection data from 150 stations that all had consecutive annual surveys between 1993 and 2014. Stations were selected that were within 0 to 16 years of LiDAR acquisition date (296 stations and \textgreater14000 individual point counts). Point count locations were spaced \(\approx\) 200 m apart.

To reduce the likelihood of double-counting birds or incorrect assignment of observations to site type, we restricted analyses to detections within the 50 m sampling radius.''

We performed our time lag analysis on seven bird species associated with different nesting and foraging guilds, forest strata, habitat structures, and forest age classes (TABLE). \citep{EltonTraits2021}.

Species were selected that exhibited low variability in the total number of detections each year across the 16 years modelled (CV \textless{} 0.5). To ensure that the selected species was abundant enough to model, we chose species that were detected in \textgreater{} 10\% of all point count events. Variability of detections between years

The seven selected species included early successional specialists: mature forests species, habitat generalist \ldots{} American Redstart (\emph{Setophaga ruticilla}), Black-throated Green Warbler (\emph{Setophaga virens}), and Swainson's Thrush (\emph{Catharus ustulatus}), Mourning Warbler (\emph{Geothlypis philadelphia}), White-throated Sparrow (\emph{Zonotrichia albicollis}), , Winter Wren(\emph{Troglodytes hiemalis}).

Based on this, we selected Black-throated Green Warbler (BTNW), a mature forest species; Swainson's Thrush (SWTH) a forest generalist; and Mourning Warbler (MOWA) an early-seral specialist. YBSA Yellow-bellied Sapsucker AMRE American redstart (mid seral) WTSP White throated sparrow (early)

To minimize the influence of forest edges and adjacent differently aged forest, we excluded point count stations with high variation of forest age classes within a hundred meter radius (SD \textgreater5 yrs)

We matched point count data from the BAM database with LiDAR data collected by the provincial government of Alberta (GOA) between 2008 and 2009, classified landcover data from CAS-FRI {[}REF{]}, and climate data from \_.

LiDAR data covering the study area collected between 2008-2009 was supplied by Alberta Agriculture and Forestry, Government of Alberta. The data was part of a wider effort to gather wall-to-wall LiDAR coverage across the province. For an overview of the LiDAR specifications and collection protocols see Alberta Environment and Sustainable Resource Development -\citet{AESRD2013}.

We selected model covariates from 34 lidar candidate predictors associated with canopy closure, height, and vegetation density. Metrics were summarized within an 100 meter radius of each point count station location using the \emph{raster} package in R \citep{R-raster}.

``CAS-FRI is a compilation of standardized forest resource inventory data from across Canada. It includes common vegetation and disturbance attributes used in bird habitat models, including forest composition and disturbance history \citep{Cumming2011a}. We used the following CAS-FRI attributes as model covariates in our analysis: post-harvest forest type (coniferous, deciduous, or mixed-wood stand), vegetation composition, crown closure, canopy height, and harvest intensity.''{[}self{]} Estimated forest age at the time of the point count, and disturbance history. As there wasn't much variation in dominant vegetation species of survey locations, we excluded vegetation composition as a covariate in our models.

Forest age, forest age class, disturbance history.

Forest age classes adapted from \citet{chen2002dynamics}.

To minimize the influence of forest edges and adjacent differently aged forest, remove stations that are surrounded by a variety of forest ages (sd\textgreater5).

We developed SDMs following the methodology outlined by Guisan and Thuiller -\citet{Guisan2005}. Using point counts conducted the same year as the LiDAR acquisition date, we used mixed effect logistic regression to assess the influence of candidate lidar predictors on specifics occupancy using the \emph{lme4} package in R\citep{batesFittingLinearMixedeffects2015} using station location as a random effect. We built global models using all available LiDAR predictors as fixed effects. We built candidate mixed effect logistic regression models useing all available lidar predictors n=34. To avoid multicollinearity between predictors we used Pearson correlation and VIF scores to iteratively removed highly correlated predictors from our models. We kept metrics with low correlation (\emph{r} \textless{} 0.5 and VIF \textless{} 3) and are associated with different measures of vegetation structure: height, cover, and structural complexity \citep{valbuenaStandardizingEcosystemMorphological2020}. For correlated metrics associated with similar, we selected the most statistically significant predictor.
Once we had a global model wth a reduced set of candidate predictors variables, deselected the best model using the \emph{MuMIn} package in R \citet{bartonMuMInMultimodelInference2020}.

I will combine LiDAR metrics with the CAS-FRI data to model species habitat relationships for American Redstart, Black-throated Green Warbler, and Swainson's Thrush. I will evaluate bird abundance as a function of LiDAR metrics and CAS-FRI data \citep{MacKenzie2006} for each year (time lag) for 10 years after the LiDAR acquisition date using GLMs. To accommodate the influence of survey methods and nuisance parameters on detection probabilities, I will generate statistical offsets via QPAD \citep{SolymosMatsuoka2013}.

While AUCs dididnt significantly decline for all species, the mapped spatial distribution did, suggesting that timelag plays a role.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\vspace{6pt} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% optional
%\supplementary{The following supporting information can be downloaded at:  \linksupplementary{s1}, Figure S1: title; Table S1: title; Video S1: title.}

% Only for the journal Methods and Protocols:
% If you wish to submit a video article, please do so with any other supplementary material.
% \supplementary{The following supporting information can be downloaded at: \linksupplementary{s1}, Figure S1: title; Table S1: title; Video S1: title. A supporting video article is available at doi: link.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\acknowledgments{This research is part of the Boreal Ecosystem Recovery and Assessment (BERA) project (\href{http://www.bera-project.org}{www.bera-project.org}), and was supported by a Natural Sciences and Engineering Research Council of Canada Alliance Grant (ALLRP 548285 - 19) in conjunction with Alberta-Pacific Forest Industries, Alberta Biodiversity Monitoring Institute, Canadian Natural Resources Ltd., Cenovus Energy, ConocoPhillips Canada, Imperial Oil Ltd., and Natural Resources Canada. This research is also part of the Boreal Avian Modelling (BAM) Project (\href{https://borealbirds.ca/}{borealbirds.ca}). We are thankful to Chris Bater for providing summarized LiDAR metrics, Greg McDermid for helping conceptualize this project, and all of the Bioacoustic Unit project leads who gathered and processed bird data.}

\authorcontributions{Conceptualization, B.C. and E.B.; methodology and formal analysis, B.C.; writing---original draft preparation, B.C.; writing---review and editing, B.C. and E.B; funding acquisition, B.C. and E.B. All authors have read and agreed to the published version of the manuscript.}

% \authorcontributions{For research articles with several authors, a short paragraph specifying their individual contributions must be provided. The following statements should be used ``Conceptualization, X.X. and Y.Y.; methodology, X.X.; software, X.X.; validation, X.X., Y.Y. and Z.Z.; formal analysis, X.X.; investigation, X.X.; resources, X.X.; data curation, X.X.; writing---original draft preparation, X.X.; writing---review and editing, X.X.; visualization, X.X.; supervision, X.X.; project administration, X.X.; funding acquisition, Y.Y. All authors have read and agreed to the published version of the manuscript.'', please turn to the  \href{http://img.mdpi.org/data/contributor-role-instruction.pdf}{CRediT taxonomy} for the term explanation. Authorship must be limited to those who have contributed substantially to the work~reported.}


\funding{}

% \funding{Please add: ``This research received no external funding'' or ``This research was funded by NAME OF FUNDER grant number XXX.'' and  and ``The APC was funded by XXX''. Check carefully that the details given are accurate and use the standard spelling of funding agency names at \url{https://search.crossref.org/funding}, any errors may affect your future funding.}
% 
\institutionalreview{Not applicable}
% 
\informedconsent{Not applicable}
% 
% \dataavailability{In this section, please provide details regarding where data supporting reported results can be found, including links to publicly archived datasets analyzed or generated during the study. Please refer to suggested Data Availability Statements in section ``MDPI Research Data Policies'' at \url{https://www.mdpi.com/ethics}. If the study did not report any data, you might add ``Not applicable'' here.} 



\conflictsofinterest{The authors declare no conflict of interest.} 

%% Optional
% \sampleavailability{Samples of the compounds ... are available from the authors.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Only for journal Encyclopedia
%\entrylink{The Link to this entry published on the encyclopedia platform.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Optional
\abbreviations{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% %% Optional
% \appendixtitles{no} % Leave argument "no" if all appendix headings stay EMPTY (then no dot is printed after "Appendix A"). If the appendix sections contain a heading then change the argument to "yes".
% \appendixstart
% \appendix
% \section[\appendixname~\thesection]{}
% \subsection[\appendixname~\thesubsection]{}
% The appendix is an optional section that can contain details and data supplemental to the main text---for example, explanations of experimental details that would disrupt the flow of the main text but nonetheless remain crucial to understanding and reproducing the research shown; figures of replicates for experiments of which representative data are shown in the main text can be added here if brief, or as Supplementary Data. Mathematical proofs of results not central to the paper can be added as an appendix.
% 
% \begin{table}[H] 
% \caption{This is a table caption.\label{tab5}}
% \newcolumntype{C}{>{\centering\arraybackslash}X}
% \begin{tabularx}{\textwidth}{CCC}
% \toprule
% \textbf{Title 1}	& \textbf{Title 2}	& \textbf{Title 3}\\
% \midrule
% Entry 1		& Data			& Data\\
% Entry 2		& Data			& Data\\
% \bottomrule
% \end{tabularx}
% \end{table}
% 
% \section[\appendixname~\thesection]{}
% All appendix sections must be cited in the main text. In the appendices, Figures, Tables, etc. should be labeled, starting with ``A''---e.g., Figure A1, Figure A2, etc.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{adjustwidth}{-\extralength}{0cm}
%\printendnotes[custom] % Un-comment to print a list of endnotes

\reftitle{References}

% Please provide either the correct journal abbreviation (e.g. according to the “List of Title Word Abbreviations” http://www.issn.org/services/online-services/access-to-the-ltwa/) or the full name of the journal.
% Citations and References in Supplementary files are permitted provided that they also appear in the reference list here. 

%=====================================
% References, variant A: external bibliography
%=====================================
\bibliography{library.bib}


% % % % \bibliography{library.bib}
% % % 



 % option sectionbib is for optionally organizing references using sections (author request)

% \ifthenelse{\equal{\@journal}{admsci}
% \OR \equal{\@journal}{arts}
% \OR \equal{\@journal}{econometrics}
% \OR \equal{\@journal}{economies}
% \OR \equal{\@journal}{genealogy}
% \OR \equal{\@journal}{humanities}
% \OR \equal{\@journal}{ijfs}
% \OR \equal{\@journal}{jrfm}
% \OR \equal{\@journal}{languages}
% \OR \equal{\@journal}{laws}
% \OR \equal{\@journal}{religions}
% \OR \equal{\@journal}{risks}
% \OR \equal{\@journal}{socsci}}{%
% 	\bibliographystyle{chicago2}
% 	\bibpunct{(}{)}{;}{x}{}{}%
% 	}{%
% 	\bibliographystyle{mdpi}
% 	\bibpunct{[}{]}{,}{n}{}{}%
% 	}%
% 
% \renewcommand\NAT@set@cites{%
%   \ifNAT@numbers
%     \ifNAT@super \let\@cite\NAT@citesuper
%        \def\NAT@mbox##1{\unskip\nobreak\textsuperscript{##1}}%
%        \let\citeyearpar=\citeyear
%        \let\NAT@space\relax
%        \def\NAT@super@kern{\kern\p@}%
%     \else
%        \let\NAT@mbox=\mbox
%        \let\@cite\NAT@citenum
%        \let\NAT@space\relax
%        \let\NAT@super@kern\relax
%     \fi
%     \let\@citex\NAT@citexnum
%     \let\@biblabel\NAT@biblabelnum
%     \let\@bibsetup\NAT@bibsetnum
%     \renewcommand\NAT@idxtxt{\NAT@name\NAT@spacechar\NAT@open\NAT@num\NAT@close}%
%     \def\natexlab##1{}%
%     \def\NAT@penalty{\penalty\@m}%
%   \else
%     \let\@cite\NAT@cite
%     \let\@citex\NAT@citex
%     \let\@biblabel\NAT@biblabel
%     \let\@bibsetup\NAT@bibsetup
%     \let\NAT@space\NAT@spacechar
%     \let\NAT@penalty\@empty
%     \renewcommand\NAT@idxtxt{\NAT@name\NAT@spacechar\NAT@open\NAT@date\NAT@close}%
%     \def\natexlab##1{##1}%
%   \fi}
% 

% 
% %% To have the possibility to change the urlcolor
% \newcommand{\changeurlcolor}[1]{\hypersetup{urlcolor=#1}} 




%=====================================
% References, variant B: internal bibliography
%=====================================
% \begin{thebibliography}{999}
% % Reference 1
% \bibitem[Author1(year)]{ref-journal}
% Author~1, T. The title of the cited article. {\em Journal Abbreviation} {\bf 2008}, {\em 10}, 142--149.
% % Reference 2
% \bibitem[Author2(year)]{ref-book1}
% Author~2, L. The title of the cited contribution. In {\em The Book Title}; Editor 1, F., Editor 2, A., Eds.; Publishing House: City, Country, 2007; pp. 32--58.
% % Reference 3
% \bibitem[Author3(year)]{ref-book2}
% Author 1, A.; Author 2, B. \textit{Book Title}, 3rd ed.; Publisher: Publisher Location, Country, 2008; pp. 154--196.
% % Reference 4
% \bibitem[Author4(year)]{ref-unpublish}
% Author 1, A.B.; Author 2, C. Title of Unpublished Work. \textit{Abbreviated Journal Name} year, \textit{phrase indicating stage of publication (submitted; accepted; in press)}.
% % Reference 5
% \bibitem[Author5(year)]{ref-communication}
% Author 1, A.B. (University, City, State, Country); Author 2, C. (Institute, City, State, Country). Personal communication, 2012.
% % Reference 6
% \bibitem[Author6(year)]{ref-proceeding}
% Author 1, A.B.; Author 2, C.D.; Author 3, E.F. Title of presentation. In Proceedings of the Name of the Conference, Location of Conference, Country, Date of Conference (Day Month Year); Abstract Number (optional), Pagination (optional).
% % Reference 7
% \bibitem[Author7(year)]{ref-thesis}
% Author 1, A.B. Title of Thesis. Level of Thesis, Degree-Granting University, Location of University, Date of Completion.
% % Reference 8
% \bibitem[Author8(year)]{ref-url}
% Title of Site. Available online: URL (accessed on Day Month Year).
% \end{thebibliography}

% If authors have biography, please use the format below
%\section*{Short Biography of Authors}
%\bio
%{\raisebox{-0.35cm}{\includegraphics[width=3.5cm,height=5.3cm,clip,keepaspectratio]{Definitions/author1.pdf}}}
%{\textbf{Firstname Lastname} Biography of first author}
%
%\bio
%{\raisebox{-0.35cm}{\includegraphics[width=3.5cm,height=5.3cm,clip,keepaspectratio]{Definitions/author2.jpg}}}
%{\textbf{Firstname Lastname} Biography of second author}

% For the MDPI journals use author-date citation, please follow the formatting guidelines on http://www.mdpi.com/authors/references
% To cite two works by the same author: \citeauthor{ref-journal-1a} (\citeyear{ref-journal-1a}, \citeyear{ref-journal-1b}). This produces: Whittaker (1967, 1975)
% To cite two works by the same author with specific pages: \citeauthor{ref-journal-3a} (\citeyear{ref-journal-3a}, p. 328; \citeyear{ref-journal-3b}, p.475). This produces: Wong (1999, p. 328; 2000, p. 475)

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% for journal Sci
%\reviewreports{\\
%Reviewer 1 comments and authors’ response\\
%Reviewer 2 comments and authors’ response\\
%Reviewer 3 comments and authors’ response
%}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{adjustwidth}
\end{document}
